src/Nethermind/Nethermind.Runner/configs/joc-mainnet.cfg

```diff
@@ -0,0 +1,32 @@
+{
+    "Init": {
+        "ChainSpecPath": "chainspec/joc-mainnet.json",
+        "GenesisHash": "0x1b54bfa6846a13aacc57066840ec10d1b74b06870539bbc8a3d1b19bdc566733",
+        "BaseDbPath": "nethermind_db/joc-mainnet",
+        "LogFileName": "joc-mainnet.logs.txt"
+    },
+    "Sync": {
+        "FastSync": true,
+        "SnapSync": true,
+        "FastBlocks": true
+    },
+    "Metrics": {
+        "NodeName": "Joc"
+    },
+    "Blocks": {
+        "TargetBlockGasLimit": 30000000
+    },
+    "JsonRpc": {
+        "Enabled": true,
+        "Timeout": 20000,
+        "Host": "127.0.0.1",
+        "Port": 8545,
+        "EngineHost": "127.0.0.1",
+        "EnginePort": 8551,
+        "EngineEnabledModules": "net,eth,subscribe,engine,web3,client,debug,trace,parity"
+    },
+    "Merge": {
+        "Enabled": true,
+        "FinalTotalDifficulty": "1"
```

Comment:
are you sure it is PoS network without difficulty increase? I think it is clique and we shouldn't have FinalTotalDifficulty and Merge.Enabled false